### ISSUE TITLE
firestore-send-email login

### ISSUE THREAD

**Issue by clecio81** [firebase/extensions#1018]
[firestore-send-email]
it was working normally at first, but now it's giving login error

Error: Invalid login: 535-5.7.8 Username and Password not accepted. Learn more at535 5.7.8  https://support.google.com/mail/?p=BadCredentials z22-20020a056870385600b0010200e2828fsm218517oal.30 - gsmtp

[SMTP]
smtps://username@gmail.com:password@smtp.gmail.com:465

**Comment by clecio81** [comment]
> Hi @clecio81
> 
> Can you confirm that the email address is still valid when authenticating through gmail?
> 
> And that correct third party access has is valid for this address via https://support.google.com/a/answer/9003945?hl=en
Yes it is working normally, I think I found the reason for the error.
![Screenshot_20220621-122338](https://user-images.githubusercontent.com/55952484/174837642-cfd28d17-33f6-47ce-8403-bef80a6b89fa.png)

Could this be the reason for the error?

**Comment by clecio81** [comment]
> Hi @clecio81
> 
> Can you confirm that the email address is still valid when authenticating through gmail?
> 
> And that correct third party access has is valid for this address via https://support.google.com/a/answer/9003945?hl=en

Yes it is working normally, is this the error? 

![IMG_20220621_122350](https://user-images.githubusercontent.com/55952484/174837822-9e83262d-1cc2-4318-a3c7-8491efd77d1e.jpg)

**Comment by dackers86** [comment]
Thanks @clecio81 

These a just initial steps for checking authentication with Google.

The next step, Google has deprecated apps on 30th May 2022. Could you also confirm that you have followed these steps:

https://support.google.com/accounts/answer/6010255?hl=en
